polaris-react/src/components/ActionList/components/SearchField/SearchField.scss

```diff
@@ -31,18 +28,10 @@ $search-icon-width-se23: calc(#{$icon-size-se23} + var(--p-space-3));
 .focused .Input,
 .Input:focus {
   border: none;
-  color: var(--p-color-text);
-
-  #{$se23} & {
-    color: var(--p-color-text-inverse);
-  }
+  color: var(--p-color-text-inverse);
```

Comment:
After testing in storybook, I'm wondering if this is the right colour. It looks like the same colour of the placeholder which is a bit odd for text

<img width="271" alt="Screenshot 2023-08-28 at 1 41 57 PM" src="https://github.com/Shopify/polaris/assets/3555826/a386c32e-6bc5-4d56-9fb4-21af2a6d645d">

Reply:
I noticed that too but since this will be replaced with TextField in another PR I think it's ok to merge with the existing color.